var/spack/repos/builtin/packages/xfsdump/package.py

```diff
@@ -39,6 +39,7 @@ def install(self, spec, prefix):
              'MSGFMT={0}'.format(self.spec['gettext'].prefix.bin.msgfmt),
              'MSGMERGE={0}'.format(self.spec['gettext'].prefix.bin.msgmerge),
              'XGETTEXT={0}'.format(self.spec['gettext'].prefix.bin.xgettext),
+             'PKG_ROOT_SBIN_DIR={0}'.format(self.spec.prefix),
```

Comment:
This causes the tools to be installed into the root prefix. However, `self.spec.prefix.sbin` also does not work. Maybe there is a better solution?